ui/app/AppLayouts/Wallet/views/NetworkSelectorView.qml

```diff
@@ -0,0 +1,143 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQml 2.15
+import QtQml.Models 2.15
+
+import StatusQ 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+
+import utils 1.0
+
+import "../controls"
+
+StatusListView {
+    id: root
+    /**
+      Expected model structure:
+        chainName      [string]          - chain long name. e.g. "Ethereum" or "Optimism"
+        chainId        [int]             - chain unique identifier
+        iconUrl        [string]          - SVG icon name. e.g. "network/Network=Ethereum"
+    **/
+    property var flatNetworks
```

Comment:
It's a list, so for users should be natural that it takes model so I would stay with just regular `model` property here.